ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.19-periodics.yaml

```diff
@@ -1126,14 +1118,6 @@ periodics:
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   minimum_interval: 72h
   name: periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-openstack-aws-conformance
-  reporter_config:
```

Comment:
We should not be deleting slack notification for the periodic jobs. Can you add them back?